.drone.yml

```diff
@@ -17,7 +17,7 @@ pipeline:
       - go vet $(go list ./... | grep -v /vendor)
       - go test -cover $(go list ./... | grep -v /vendor)
       - go build $(go list ./... | grep -v /vendor)
-      - go build github.com/${DRONE_REPO}/cmd/kmm
+      - CGO_ENABLED=0 go build -a -ldflags -v github.com/${DRONE_REPO}/cmd/kmm
```

Comment:
Given your not passing a linker parameters on -ldflags you could just drop it as it's a noop